GI05MOAS-GL525/GI05MOAS-GL525_D00003_ingest.csv

```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL525/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL525-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flord_m.flord_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL525/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL525-01-FLORDM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL525/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL525-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL525/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL525-04-CTDGVM000,telemetered,Available,
```

Comment:
looks good but just want to verify that the glider # does not need to be inserted at end of filename_mask (merged-from-glider/gi_525*.mrg?  Some older ingest csvs have it, but maybe that was determined to be unnecessary?